src/AspNetIdentity/build/build.csproj

```diff
@@ -5,13 +5,12 @@
     <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Compile Include="..\..\build\Program.Partial.cs" Link="Program.Partial.cs" />
+  <ItemGroup>
+    <Compile Include="..\..\build\Program.Partial.cs" Link="Program.Partial.cs" />
```

Comment:
The line endings were wrong on these two lines (CRLF instead of LF), and I can't figure out a way to get an editor to preserve them when making the change below.

This is part of a more general problem, in that `.editorconfig` has `end_of_line = crlf`, but almost every file in the repo has LF, which suggests that local `autocrlf` settings have been relied upon to control line endings. But this is a can of worms best opened another time...